contract/contract.go

```diff
@@ -21,6 +21,18 @@ var AutoGenWhiteList = map[string]bool{
 	"kernel.UpdateMaxBlockSize":  true,
 }
 
+// UtxoMetaRegister in avoid to being refered in a cycle way
+type UtxoMetaRegister interface {
```

Comment:
Is it better to wrap all genesis configs in a struct, and make the struct one member of UTXOMeta?
By naming UtxoMetaRegister, it's more like a register for all UTXOMeta members, but it's just for genesis configs.

Reply:
Thank you for your suggestions. And I plan to give a reasonable name instead of UtxoMetaRegister and all the genesis configures have been the members of UtxoMeta.